recipe/install-liblief.bat

```diff
@@ -1,4 +1,2 @@
-pushd build
```

Comment:
This needs to be restored as the project is configured inside a folder called build.

Reply:
I´m undoing the build folder. Missed this one.